src/main/java/de/php_perfect/intellij/ddev/version/GithubClient.java

```diff
@@ -19,7 +19,7 @@ public final class GithubClient implements ReleaseClient {
 
     @Override
     public @Nullable LatestRelease loadCurrentVersion(@NotNull ProgressIndicator indicator) {
-        final RequestBuilder requestBuilder = HttpRequests.request(RELEASE_URL).accept("application/json").redirectLimit(1);
+        final RequestBuilder requestBuilder = HttpRequests.request(RELEASE_URL).accept("application/json").redirectLimit(2);
```

Comment:
Did you find out why the limited needs to be increased?

Reply:
I did not look into the history in detail, but this commit from 8 months ago looks like the most probable cause: https://github.com/JetBrains/intellij-community/commit/b793719aac02659cef47aad371873bff0a15f51f

Reply:
Okay, looks like the additional redirect of github was ignored before this commit, yes.
Next thing is to find out why that redirect is happening, but not relevant for this pull request.
Will merge it, as soon as the tests are stable. :)

Reply:
There is only one redirect from latest to the current version, no additional redirects.
```
private static URLConnection openConnection(RequestBuilderImpl builder, RequestImpl request) throws IOException {
    if (builder.myForceHttps && request.myUrl.startsWith("http:")) {
      request.myUrl = "https:" + request.myUrl.substring(5);
    }

    for (int i = 0; i < builder.myRedirectLimit; i++) {
```

Unless I'm somehow reading this super wrong, the with a theoretical myRedirectLimit of 0 (which can't be set, the minimum is 1) you'd not get any connection.
You need at least 1 to get a connection to a page without redirects, and 2 if there is 1 redirect. Always 1 + expected redirects.

Reply:
Ahh, I love off by one errors! :D
Thanks!